### PR TITLE
dbus crate replaced with zbus

### LINKED ISSUE #148: Replace the dbus crate with zbus
To get rid of the dependency to the native libdbus, alternatives to the dbus crate were evaluated in #139. zbus was chosen as a good replacement for the dbus crate.

### PR DESCRIPTION
**Pull request by siegfriedweber**
Closes #148

### CODE REVIEW

**Review by soenkeliebau — DISMISSED** [review]
LGTM - left a few minor comments, but nothing that would prohibit this from being merged.
Ran integration tests those were successfull as well.

File: Cargo.lock [hunk 1]
```diff
@@ -52,23 +52,11 @@ version = "1.0.3"
...
-[[package]]
-name = "arrayvec"
-version = "0.5.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "23b62fc65de8e4e7f52534fb52b0f3ed04746ae267519eef2a83941e8085068b"
-
 [[package]]
 name = "async-compression"
-version = "0.3.7"
+version = "0.3.8"
```
[the remark this answers is not in the document]
Comment (siegfriedweber) [reply Cargo.lock:57]: This was on purpose. There were several dependabot updates which forced me to rebase this branch and fix conflicts so I decided to just update everything and test the changes extensively.

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -15,15 +15,15 @@ jobs:
...
+    - run: sudo apt-get install libsystemd-dev pkg-config
     - run: rustup component add rustfmt
     - run: cargo fmt --all -- --check
 
   clippy_check:
     runs-on: ubuntu-latest
     steps:
-      - name: Install dbus dependencies
-        run: sudo apt-get install libdbus-1-dev libsystemd-dev pkg-config libdbus-1-3
+      - name: Install systemd dependencies
```
[the remark this answers is not in the document]
Comment (siegfriedweber) [reply .github/workflows/rust.yml:25]: pkg-config is necessary for libsystemd-sys and openssl-sys. But it is a good idea to give the step a more general name.

**Review by soenkeliebau — APPROVED** [review]
LGTM

**Pull request merged**